core/generated.go

```diff
@@ -116,6 +118,9 @@ type GenerateProps struct {
 
 	// The target type - must be either "host" or "target"
 	Target tgtType
+
+	// If true, depfile name will be generated and can be used as $depfile reference in 'cmd'
```

Comment:
Can you change `$depfile` to `${depfile}` in this comment please? That's what we check for in the error message above.